auto_updater.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash set -eu
 
+PUSH=
```

Comment:
not defaulting to false as opposed to undefined?

Reply:
comment what that is

Reply:
https://github.com/AdoptOpenJDK/homebrew-openjdk/pull/243/files#diff-74755599760817973315cd18477be295R8